build.sh

```diff
@@ -14,52 +15,72 @@ echo "SNAPSHOT ${AURORA_SNAPSHOT}"
 echo "DOCKER IMAGE ${AURORA_IMAGE}"
 
 BASE_DIR="$(pwd)"
+BUILD_PACKAGE=${BUILD_PACKAGE:-1}
 
-if [ ! -f aurora-scheduler_${AURORA_RELEASE}_amd64.deb ]; then
+function upload {
+  if [ ! -f $1 ]; then
+    echo "Cannot find package $1"
+    exit 1
+  fi
+  STATUS=""
+  echo "Uploading $1 to github"
+  UPLOAD_URL="${UPLOAD_URL}?name=$1" 
+  RES=$(curl -# -XPOST -H "Authorization:token ${GITHUB_TOKEN}" -H "Content-Type:application/octet-stream" --data-binary @$1 ${UPLOAD_URL})
+  if [[ -z $(echo "$RES" | grep -o "\"state\":\"[a-z]*") ]]; then
+    echo "Upload failed"
+    echo "Is the package already uploaded ? Delete first"
+    echo "$RES" | grep -o 'message\":\"[A-Za-z ]*'\"
+    exit 1
+  fi
+}
+
+if [ $BUILD_PACKAGE -eq 1 ]; then
   # fetch source
   curl -L -o "snap.tar.gz" "$AURORA_SNAPSHOT"
+
+  # XXX: Without this aurora-packaging fails for .rpms since it expects 
+  # something of the form apache-aurora-xxx
+  tar -xvzf snap.tar.gz 
+  mv aurora-rel-$AURORA_VERSION-medallia apache-aurora-$AURORA_VERSION
+  tar -czvf snap.tar.gz apache-aurora-$AURORA_VERSION 
+  rm -rf apache-aurora-$AURORA_VERSION
+
   if [ ! -d aurora-packaging ]; then
       git clone "https://github.com/apache/aurora-packaging.git" "aurora-packaging"
   fi
 
   (cd aurora-packaging && \
-   git fetch origin "$AURORA_PACKAGE_BRANCH" && \
-   git checkout "$AURORA_PACKAGE_BRANCH" && \
-   ./build-artifact.sh builder/deb/ubuntu-trusty \
+  git fetch origin "$AURORA_PACKAGE_BRANCH" && \
```

Comment:
optional: the (cd ...) pattern is getting hard to read now, what about switching to `pushd aurora-packaging`, execute everything and then `popd`. We already set -e so any error would stop the script, so the `&&` are not really needed